src/ShoppingListForm.js

```diff
@@ -1,13 +1,20 @@
 import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
 import { db } from './lib/firebase.js';
 import './ShoppingListForm.css';
 
 function ShoppingListForm() {
   const [item, setItem] = useState('');
+  const [buyTime, setBuyTime] = useState(7);
+  const [lastPurchased, setLastPurchased] = useState(null);
+  const [token, setToken] = useState(localStorage.getItem('userToken') || '');
 
   const handleSubmit = (event) => {
-    db.collection('shoppingLists').add({
-      items: [item],
+    db.collection('items').add({
```

Comment:
This might be better as an array of items attached to the lists collection. We can also attach the token to the lists collection. This solves the AC in 3. "For users who do not already have a token/list, a button or link exists on the home screen that allows them to create a new list"

We can query for the token on list to return the full set of items

Reply:
Thinking about it, we should be fine keeping the token on the item itself.